### PR TITLE
update README

### LINKED ISSUE #3469: updating the README.md
I think the 1.0 release is a good opportunity to give the rather outdated README.md a bit of a polish.

I plan to:
 - [x] add the AiiDA logo
 - [x] borrow the short description of features from the docs
 - [x] add a number of badges
    - [x] conda-forge
    - [x] license
    - [x] pypi stats
    - [ ] ~~feature request issues vs bug issues (?)~~
    - [ ] ~~percentage of open issues (?)~~
    - [x] mailing list
    - [ ] ~~code quality~~

### PR DESCRIPTION
**Pull request by ltalirz**
fix #3469 
 * Add AiiDA logo
 * Add list of features
 * Add badges:
   * supported python versions
   * conda forge releases
   * Google group
   * license
   * pypi downloads/month, commit activity
   * open issues %
   * bug issues
   * issue time to resolution
   * good first issue issues
   * PRs welcome

Note: I'm removing the note on the open source licenses from the license file (since it prevents it from being recognized as MIT), and rather mention it in the README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,69 @@
...
+
+|    | |
+|-----|----------------------------------------------------------------------------|
+|Latest release| [![PyPI version](https://badge.fury.io/py/aiida-core.svg)](https://badge.fury.io/py/aiida-core) [![conda-forge](https://img.shields.io/conda/vn/conda-forge/aiida-core.svg?style=flat)](https://anaconda.org/conda-forge/aiida-core) [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-core.svg)](https://pypi.python.org/pypi/aiida-core/) |
+|Getting help| [![Docs status](https://readthedocs.org/projects/aiida-core/badge)](http://aiida-core.readthedocs.io/) [![Google Group](https://img.shields.io/badge/-Google%20Group-lightgrey.svg)](https://groups.google.com/forum/#!forum/aiidausers)
+|Build status| [![Build Status](https://travis-ci.org/aiidateam/aiida-core.svg?branch=develop)](https://travis-ci.org/aiidateam/aiida-core) [![Coverage Status](https://coveralls.io/repos/github/aiidateam/aiida-core/badge.svg?branch=develop)](https://coveralls.io/github/aiidateam/aiida-core?branch=develop) |
+|License| [![License](https://img.shields.io/github/license/aiidateam/aiida-core.svg)](https://github.com/aiidateam/aiida-core/blob/develop/LICENSE.txt)|
```
Comment (sphuber) [inline README.md]: License shows as not identifiable by Github. Do we know why and can we fix it?

File: README.md [hunk 2]
```diff
@@ -1,68 +1,69 @@
...
+|Activity| [![PyPI-downloads](https://img.shields.io/pypi/dm/aiida-core.svg?style=flat)](https://pypistats.org/packages/aiida-core) [![HitCount](http://hits.dwyl.io/aiidateam/aiida-core.svg)](http://hits.dwyl.io/aiidateam/aiida-core) [![Commit Activity](https://img.shields.io/github/commit-activity/m/aiidateam/aiida-core)](https://github.com/aiidateam/aiida-core/pulse)
+|Development| [![Percentage of issues still open](https://isitmaintained.com/badge/open/aiidateam/aiida-core.svg)](https://isitmaintained.com/project/aiidateam/aiida-core "Percentage of issues still open") [![Bugs](https://img.shields.io/github/issues/aiidateam/aiida-core/type%2Fbug.svg)](https://github.com/aiidateam/aiida-core/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%2Fbug%22+sort%3Areactions-%2B1-desc)   [![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/aiidateam/aiida-core.svg)](http://isitmaintained.com/project/aiidateam/aiida-core "Average time to resolve an issue")|
```
Comment (sphuber) [inline README.md]: commit activity badge shows as invalid, is this fixable or if not better maybe to remove it?

File: README.md [hunk 3]
```diff
@@ -1,68 +1,69 @@
...
 
-License
--------
+## License
+
+AiiDA is distributed under the MIT open source license (see [`LICENSE.txt`](LICENSE.txt)).
 
-The terms of the AiiDA license can be found in the LICENSE.txt file.
+For a list of other open source components included in AiiDA, see the
+file [`open_source_licenses.txt`](open_source_licenses.txt).
```
Comment (sphuber) [inline README.md]: This file is outdated and there are references to files that no longer exist. We should really update it if we're going to keep it and link it

File: README.md [hunk 4]
```diff
@@ -1,68 +1,69 @@
...
 
 This work is supported by the [MARVEL National Centre for Competency in
 Research](<http://nccr-marvel.ch>) funded by the [Swiss National
 Science Foundation](<http://www.snf.ch/en>), as well as by the [MaX
 European Centre of Excellence](<http://www.max-centre.eu/>) funded by
 the Horizon 2020 EINFRA-5 program, Grant No. 676598.
 
-![MARVEL](docs/source/images/MARVEL.png)
-![MaX](docs/source/images/MaX.png)
+<img src="docs/source/images/MARVEL.png" alt="AiiDA" width="120"/>
```
Comment (sphuber) [inline README.md]: Did you change this explicitly to get the images to be vertically stacked? Would horizontal be better maybe, although with some decent margins

File: README.md (hunk 1 above)
Comment (ltalirz) [reply README.md]: I think it has to do with the addition at the bottom of the license.
This PR moves the addition from there to the README

File: README.md (hunk 2 above)
Comment (ltalirz) [reply README.md]: Ok, we need to fix this - for some reason, when I paste the link to the svg in the browser it works (so I thought it was an issue of some cache not working).
I'll look into it.

File: README.md (hunk 3 above)
Comment (ltalirz) [reply README.md]: Thanks, I'll have a look

File: README.md (hunk 4 above)
Comment (ltalirz) [reply README.md]: Agreed - somehow github renders this differently than my markdown viewer.
I'll look into it.

**Comment by ltalirz** [comment]
> The top of the page looks a bit like a Christmas tree now :)

I like the badges because they can give you a lot of information about a project in one glance.
I've now removed one badge and moved the license one down to the license header.
For the others, I feel they are useful to have at the top - but of course it's also a matter of taste..

@giovannipizzi What's your take?
We can also gather feedback more broadly...

**Comment by ltalirz** [comment]
For simplicity, here is the link: https://github.com/ltalirz/aiida_core/blob/update-readme/README.md

**Comment by sphuber** [comment]
Discussed in person with Giovanni and what about the following:
Keep the top 4 rows of the badge table
Remove the 5th about Development.
Remove the 6th about Contributing but create a small section below with the link to the contribution guide (the badges don't seem very helpful, the first is obvious and the `good-first-issue` concept we are not really using).

**Comment by ltalirz** [comment]
done!
for the moment, I moved the two badges for new people down to the contributing section.
we could also decide to move those to the wiki... but perhaps one could capture someone here more easily..

**Pull request merged**